app/concepts/user/create.rb

```diff
@@ -0,0 +1,12 @@
+class User::Create < Trailblazer::Operation
+  include Model
+  model User
+
+  def process(params)
+    @model = User.new
+    @model.fullname = "Guest"
+    @model.save
+    return @model
```

Comment:
May I know also the reason why you had to call `return @model` in here? Kindly check out this link http://trailblazer.to/ to see how `validate` is being implemented in here.